File: Programming Fundamentals/Labs And Homeworks/ArraysExercise/ArrayRotation.java
package ArraysExercise;

import java.util.Scanner;

public class ArrayRotation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] lineArray = scanner.nextLine().split(" ");
        int rotation = Integer.parseInt(scanner.nextLine());
        for (int i = 1; i <= rotation; i++) {
            String firstElement = lineArray[0];
            for (int j = 0; j < lineArray.length - 1; j++) {
                lineArray[j] = lineArray[j + 1];
            }
            lineArray[lineArray.length - 1] = firstElement;
        }
        for (int i = 0; i < lineArray.length; i++) {
            System.out.print(lineArray[i] + " ");
        }

    }
}

File: JS Front-End/JS Syntax Fundamentals - Exercise/arrayRotation.js
function arrayRotation(array, rotation) {
    while (rotation > 0) {
        let current = array.shift();
        array.push(current);
        rotation--;
    }
    console.log(array.join(" "));
}

arrayRotation([51, 47, 32, 61, 21], 2);